airflow/providers/amazon/aws/triggers/redshift_cluster.py

```diff
@@ -262,3 +264,53 @@ def __init__(
 
     def hook(self) -> AwsGenericHook:
         return RedshiftHook(aws_conn_id=self.aws_conn_id)
+
+
+class RedshiftClusterSensorTrigger(BaseTrigger):
```

Comment:
I am wondering if "sensor" is needed in the trigger name. I feel like "sensor"is implicit because the trigger will always sense something until it reaches a given state

Reply:
Sounds good. I just renamed it as `RedshiftClusterTrigger`